### PR TITLE
Timestamp when instead of at which

### PR DESCRIPTION
**Pull request by adamaltman**
## Summary

"Date and time when" seems more natural than "Date and time at which".

Thoughts?

## Checklist

- [ ] Writing style
- [ ] API design standards

### CODE REVIEW

**Review by XaosSintez — APPROVED** [review]
Agree

**Review by creocoder — APPROVED** [review]
LGTM, left alternate suggestions and some thoughts

File: WRITING-STYLE.md [hunk 1]
```diff
@@ -243,14 +243,14 @@ Coupon created time:
 
 ```yaml
   createdTime:
-    description: Date and time at which the coupon is created.
+    description: Date and time when the coupon is created.
```
Comment (creocoder) [inline WRITING-STYLE.md:246]:
```suggestion
    description: Date and time the coupon was created.
```

File: WRITING-STYLE.md [hunk 2]
```diff
@@ -243,14 +243,14 @@ Coupon created time:
...
 ```
 
 Coupon expired time:
 
 ```yaml
   expiredTime:
-    description: Date and time at which the coupon expires.
+    description: Date and time when the coupon expires.
```
[the remark this answers is not in the document]
Comment (creocoder) [reply WRITING-STYLE.md:253]: I also think original property `expiredTime` carries the wrong meaning. The name of the property implies the past tense. We should generally reconsider the naming of properties of this type. Possible good name: `expirationSheduledAt`.

File: WRITING-STYLE.md (hunk 1 above)
Comment (creocoder) [reply WRITING-STYLE.md:246]: Reason: the shortest possible definition, without loss of meaning.

Comment (adamaltman) [reply WRITING-STYLE.md:246]: This suggestion violates the future-past rule.

**Review by ashkarpetin — APPROVED** [review]

File: WRITING-STYLE.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (adamaltman) [reply WRITING-STYLE.md:253]: Property names for date-time are an issue. But unless we version or deprecate it is going to be difficult to change property names.

File: WRITING-STYLE.md (hunk 1 above)
Comment (creocoder) [reply WRITING-STYLE.md:246]: Got it, thank you

**Review by vykhrystiuk — APPROVED** [review]

**Review by noon-dawg — APPROVED** [review]

**Comment by noon-dawg** [comment]
Good point. `at which` seems like a much older and more formal way of saying it 😄

**Pull request merged**